app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -98,453 +99,447 @@ export function addComposerUploadMarkdownResolver(resolver) {
 export function cleanUpComposerUploadMarkdownResolver() {
   uploadMarkdownResolvers = [];
 }
+export default Component.extend(
+  ComposerUploadUppy,
+  ComposerVideoThumbnailUppy,
```

Comment:
Most changes are prettier indentation changes caused by adding this line. Try viewing without whitespace changes to see a better diff https://github.com/discourse/discourse/pull/19801/files?w=1